bin/pygrb/pycbc_pygrb_plot_injs_results

```diff
@@ -283,156 +255,113 @@ if not os.path.isdir(outdir):
-logging.info("Loading timeslides.")
-slide_dict = ppu.load_time_slides(trig_file)
-logging.info("Loading segments.")
-segment_dict = ppu.load_segment_dict(trig_file)
-
-# Construct trials
-logging.info("Constructing trials.")
-trial_dict = ppu.construct_trials(opts.seg_files, segment_dict,
-                                  ifos, slide_dict, vetoes)
-total_trials = sum([len(trial_dict[slide_id]) for slide_id in slide_dict])
-logging.info("%d trials generated.", total_trials)
-
-# Extract basic trigger properties and store as dictionaries
-trig_time, _, trig_bestnr = \
-    ppu.extract_basic_trig_properties(trial_dict, trigs, slide_dict, segment_dict, opts)
-
-# Calculate SNR and BestNR values and maxima
-time_veto_max_bestnr = {}
-for slide_id in slide_dict:
-    num_slide_segs = len(trial_dict[slide_id])
-    time_veto_max_bestnr[slide_id] = np.zeros(num_slide_segs)
-
-for slide_id in slide_dict:
-    for j, trial in enumerate(trial_dict[slide_id]):
-        trial_cut = (trial[0] <= trig_time[slide_id])\
-                          & (trig_time[slide_id] < trial[1])
-        if not trial_cut.any():
-            continue
-        # Max BestNR
-        time_veto_max_bestnr[slide_id][j] = \
-                        max(trig_bestnr[slide_id][trial_cut])
-
-max_bestnr, _, full_time_veto_max_bestnr = ppu.max_median_stat(slide_dict,
-                                                               time_veto_max_bestnr,
-                                                               trig_bestnr,
-                                                               total_trials)
-
-logging.info("BestNR maxima calculated.")
+trig_data = load_trig_data(trig_file, vetoes)
+max_reweighted_snr = max(trig_data['network/reweighted_snr'][:]) ## FOR TESTING
 
 # =======================
 # Post-process injections
 # =======================
 # Triggers, missed injections (i.e., not recovered at all), and injections
 # recovered in some form. Trigs/injs at vetoed times are discarded.
-found_trigs = ppu.load_injections(found_file, vetoes, label="triggers")
-missed_injs = ppu.load_injections(missed_file, vetoes, sim_table=True, label="missed injections")
-found_injs = ppu.load_injections(found_file, vetoes, sim_table=True, label="found injections")
-logging.info("Triggers and missed/found injections loaded.")
 
-# Extract the detection statistic of found triggers
-found_trig = {}
-found_trig['bestnr'] = ppu.get_bestnrs(found_trigs, q=chisq_index, n=chisq_nhigh,
-                                       null_thresh=null_thresh,
-                                       snr_threshold=snr_thresh,
-                                       sngl_snr_threshold=sngl_snr_thresh,
-                                       chisq_threshold=new_snr_thresh,
-                                       null_grad_thresh=null_grad_thresh,
-                                       null_grad_val=null_grad_val)
+# Load indices containing found and missed injections
```

Comment:
Please expand on the comments, following the structure we used to have, which was
```
# 1) indices of injections found louder than background
# 2) found (bestnr > 0) but not louder than background (non-zero FAP)
# 3) missed after being recovered (i.e., vetoed)
```
Now you have (I think)
```
found: indices of all found injections (bestnr > 0 and louder than loudest background, i.e., FAP=0)
missed: indices of injections never recovered
found_after_vetoes: indices of all found injections surviving vetoes
missed_after_vetoes: indices of all found but vetoed injections
```
As far as I can tell so far, the old group 2 is missing (but it may be populated further down in the code)

Reply:
Ok, these you call `quieter_indices` later on.